crafters/nlp/TikaExtractor/__init__.py

```diff
@@ -0,0 +1,60 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+
+import requests
+from requests.exceptions import ConnectionError
+from jina.executors.crafters import BaseCrafter
+
+TIKA_URL = 'http://0.0.0.0:9998'
+
+
+class TikaExtractor(BaseCrafter):
+    """
+    :class:`TikaExtractor` Extracts text from files.
+    """
+
+    def start_tika(self):
+        from subprocess import Popen
+        tika_version = os.getenv('TIKA_VERSION')
```

Comment:
It seems a little odd to have this depend on an env variable?

Reply:
The env variable is set by the upstream docker image